### ISSUE TITLE
Theme and Layout do not apply on startup

### ISSUE THREAD

**Issue by banksio** [Hyrules/WinHue3#182]
Known and will be fixed. Created by #181.

**Comment by banksio** [comment]
@Hyrules Apologies I haven't been able to work on theming in a while, I've been quite busy lately. I do hope to return to this though.

**Comment by Hyrules** [comment]
no probleme i havn't worked on winhue either for the past few months. i
just reverted the dev branch to pre theming so everything work as intended
you can continue working on the theming branch until it's ready meanwhile
i'll release a version soon.

**Comment by banksio** [comment]
@Hyrules Yeah, no worries. It needs some more work to polish it up before it goes out in a release.

**Comment by banksio** [comment]
This is now fixed in the latest `theming`.
